kazoo/handlers/utils.py

```diff
@@ -291,12 +305,55 @@ def wrap(async_result):
     :param async_result: An async result implementing :class:`IAsyncResult`
 
     """
+
     def capture(function):
         @capture_exceptions(async_result)
         def captured_function(*args, **kwargs):
             value = function(*args, **kwargs)
             if value is not None:
                 async_result.set(value)
             return value
+
         return captured_function
+
     return capture
+
+
+def _selector_select(rlist, wlist, xlist, timeout=None, *,
+                     selectors_module=None):
+    """selector-based drop-in replacement for select to overcome select
+    limitation on a maximum filehandle value.
+
+    Need backport selectors2 package in python 2.
+    """
+    if timeout is None:
+        timeout = -1
+    selectors_module = selectors if not selectors_module else selectors_module
```

Comment:
[nit] why not just have `selectors` as the default value for `selectors_module`?